src/main/scala/com/wavesplatform/network/PeerDatabaseImpl.scala

```diff
@@ -1,32 +1,55 @@
 package com.wavesplatform.network
 
 import java.net.{InetAddress, InetSocketAddress}
+import java.nio.charset.MalformedInputException
 
+import com.fasterxml.jackson.databind.JsonMappingException
 import com.google.common.collect.EvictingQueue
 import com.wavesplatform.settings.NetworkSettings
-import com.wavesplatform.utils.createMVStore
+import com.wavesplatform.utils.JsonFileStorage
 import io.netty.channel.Channel
 import io.netty.channel.socket.nio.NioSocketChannel
-import org.h2.mvstore.MVMap
-import scorex.utils.{LogMVMapBuilder, ScorexLogging}
+import play.api.libs.json._
+import scorex.utils.ScorexLogging
 
+import scala.collection._
 import scala.collection.JavaConverters._
 import scala.util.Random
 
 class PeerDatabaseImpl(settings: NetworkSettings) extends PeerDatabase with AutoCloseable with ScorexLogging {
 
-  private val database = createMVStore(settings.file)
-  private val peersPersistence = database.openMap("peers", new LogMVMapBuilder[InetSocketAddress, Long])
-  private val blacklist = database.openMap("blacklist", new LogMVMapBuilder[InetAddress, Long])
-  private val suspension = database.openMap("suspension", new LogMVMapBuilder[InetAddress, Long])
-  private val reasons = database.openMap("reasons", new LogMVMapBuilder[InetAddress, String])
+  implicit val inetSocketAddressRW: OFormat[InetSocketAddress] = OFormat.apply[InetSocketAddress](jsValue => {
```

Comment:
Please use custom `case class Address(hostname, port)` or whatever, `implicit val setR: AnyRef with Reads[Set[InetSocketAddress]] ` should be gone